### DISCUSSION TITLE
Hierarchical mapping of endpoints

### DISCUSSION THREAD

**Discussion by jprk** [mar10/wsgidav#283] [Q&A]
Hello,

I have recently reinstalled a crashed machine where I used wsgidav to serve some data files collected by different services and stored in different locations to MS Windows (RaiDrive) clients. I have lost the original configuration to the crash, and I am struggling a bit to tweak the configuration to do exactly what I want.

I think that I had the endpoints mapped approximately as follows:
``` yaml
provider_mapping:
    # Map the home directory of a user to /
    '/': '~'
    # Add an endpoint for service 1
    '/data_pos': '/data/pos_collector/outputs'
    # Add an endpoint for service 2
    '/data_detectors': '/data/detector_service/output'
``` 

and then, after logging into the server, I had in the initial listing of `/` also `/data_pos` and `/data_detectors` present. Is it possible? The current directory listing of `/` namely does not include the `/data_pos` and `/data_detectors` endpoints -- and I do not know if it is by design or if I forgot to configure something somewhere.

Currently, I am using a workaround by creating two empty directories `~/data_pos` and `~/data_detectors` in the home directory of users who have access, but I still think I have been doing it in a different way. But maybe I am mistaken ...

Thanks!

**Comment by mar10** [comment]
Are you sure that this works for different users? I would assume that `~` always resolves to the home folder of the WsgiDAV service.

(see #29)
